projects/dtt-app/src/app/app.component.html

```diff
@@ -0,0 +1,43 @@
+<mat-sidenav-container fullscreen>
+    <mat-sidenav position="end" mode="side" (openedChange)="mapComponent.resize()">
+        <wave-sidenav-container></wave-sidenav-container>
+    </mat-sidenav>
+    <div class="time-container" fxLayoutAlign="space-around center">
+        <wave-zoom-handles (zoomIn)="map.zoomIn()" (zoomOut)="map.zoomOut()" class="mat-elevation-z4"></wave-zoom-handles>
+        <wave-small-time-interaction class="mat-elevation-z4"></wave-small-time-interaction>
+        <wave-ticker-interaction *ngIf="config.COMPONENTS.PLAYBACK.AVAILABLE" class="mat-elevation-z4"></wave-ticker-interaction>
+        <div></div> <!-- moves Time Interaction to center (cf. space-around @ flex) -->
+    </div>
+    <wave-navigation [buttons]="navigationButtons"></wave-navigation>
+    <wave-layer-list class="mat-elevation-z4"
+                     [height]="middleContainerHeight$ | async"
+                     [addAFirstLayerSidenavConfig]="addAFirstLayerConfig">
+        <div class="logo">
+            <mat-icon mat-list-icon svgIcon="vat:logo" aria-label="VAT"></mat-icon>
+            <span class="title">DTT</span>
```

Comment:
powered by VAT or Geo Engine?
(cf. old geo bon app)

Reply:
TODO in another task

Reply:
I crated a task for it.